apps/studio/components/layouts/SQLEditorLayout/SQLEditorMenu.tsx

```diff
@@ -64,9 +66,6 @@ export const SQLEditorMenu = () => {
     if (!ref) return console.error('Project ref is required')
     if (!project) return console.error('Project is required')
     if (!profile) return console.error('Profile is required')
-    if (!canCreateSQLSnippet) {
-      return toast('Your queries will not be saved as you do not have sufficient permissions')
-    }
```

Comment:
looks to me like this line is supposed to block navigating to the new sql snippet route